### PR TITLE
Bumps in garden : ign-utils2, ign-plugin2

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/685

### CODE REVIEW

File: examples/worlds/multi_lrauv_race.sdf [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 
 <!-- Run demo with ign-gazebo examples/standalone/multi_lrauv_race
-  (example is provided in ign-gazebo6+). -->
+  (example is provided in ign-gazebo7+). -->
```
Comment (chapulina) [inline examples/worlds/multi_lrauv_race.sdf:4]: Oops

File: src/gui/plugins/spawn/Spawn.cc [hunk 2]
```diff
@@ -190,7 +190,7 @@ void Spawn::LoadConfig(const tinyxml2::XMLElement *)
 }
 
 
-// TODO(ahcorde): Replace this when this function is on ign-rendering6
+// TODO(ahcorde): Replace this when this function is on ign-rendering7
```
Comment (chapulina) [inline src/gui/plugins/spawn/Spawn.cc:193]: Unintentional change. But in any case this comment is going away once https://github.com/ignitionrobotics/ign-gazebo/pull/1432/ is forward-ported.

**Review by chapulina — APPROVED** [review]
LGTM with 🟢 CI

File: src/gui/plugins/spawn/Spawn.cc (hunk 2 above)
Comment (methylDragon) [reply src/gui/plugins/spawn/Spawn.cc]: https://github.com/ignitionrobotics/ign-gazebo/pull/1444/commits/2117f1c4ba31917ffd83064d14d1ac0ccc765dc2

File: examples/worlds/multi_lrauv_race.sdf (hunk 1 above)
Comment (methylDragon) [reply examples/worlds/multi_lrauv_race.sdf]: https://github.com/ignitionrobotics/ign-gazebo/pull/1444/commits/2117f1c4ba31917ffd83064d14d1ac0ccc765dc2

**Review by chapulina — APPROVED** [review]

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please

**Comment by chapulina** [comment]
* ✅ GitHub actions Focal only has known test failures
* ⌛ Windows is compiling Gazebo
* ⌛ Homebrew is compiling dependencies
* ⌛ Jenkins Ubuntu is running tests

**Comment by chapulina** [comment]
Ouch homebrew was so close 😢 

```
Error: Could not symlink lib/python/sdformat.cpython-39-darwin.so
Target /usr/local/lib/python/sdformat.cpython-39-darwin.so
is a symlink belonging to sdformat13. You can unlink it:
  brew unlink sdformat13

To force the link and overwrite all conflicting files:
  brew link --overwrite ignition-gazebo7

To list all files that would be deleted:
  brew link --overwrite --dry-run ignition-gazebo7
```

I believe this is not this PR's fault, but a new conflict introduced by the new SDF python bindings. CC @ahcorde 

I'm not going to block this PR on this because Garden has been broken long enough and macOS is not an officially supported platform. We'll come back to fix it promptly.

**Pull request merged**